streams/am.m3u

```diff
@@ -5,7 +5,5 @@ https://amtv1-2.livestreamingcdn.com/am3abr/index.m3u8
 https://amtvusdvr.tulix.tv/am3abr/index.m3u8
 #EXTINF:-1 tvg-id="SONGTVArmenia.am",SONGTV Armenia (720p)
 https://songtv.hls.iptvdc.com/web-armenia/index.m3u8
-#EXTINF:-1 tvg-id="SONGTVRussia.ru",SONGTV Russia (720p)
-https://songtv.hls.iptvdc.com/web-russia/index.m3u8
```

Comment:
Why? This is Armenian channel, will be easy to remove/edit in a single place when backend changes